packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -144,6 +147,92 @@ async function handleGrant(releaseGoldConfig: any, currGrant: number) {
   }
 }
 
+async function checkBalance(releaseGoldConfig: any) {
+  const weiAmountReleasedPerPeriod = new BigNumber(
+    web3.utils.toWei(releaseGoldConfig.amountReleasedPerPeriod.toString())
+  )
+  const grantTotal = weiAmountReleasedPerPeriod
+    .multipliedBy(releaseGoldConfig.numReleasePeriods)
+    .plus(startGold)
+    .toFixed()
+  while (true) {
+    const fromBalance = new BigNumber(await web3.eth.getBalance(fromAddress))
+    if (fromBalance.gte(grantTotal)) {
+      break
+    }
+    console.info(
+      chalk.yellow(
+        fromAddress +
+          "'s balance will not cover the next grant with identifier " +
+          releaseGoldConfig.identifier +
+          '.'
+      )
+    )
+    const addressResponse = await prompts({
+      type: 'text',
+      name: 'newFromAddress',
+      message:
+        'If you would like to continue deployment from a new address please provide it now. If you would like to cancel deployment just press enter. Grants already deployed will persist.',
```

Comment:
minor nit: I worry about accidentally hitting enter here, can we instead require "exit" to be typed in to exit?